mindjunkies/forums/forms.py

```
@@ -17,7 +28,12 @@ def save(self, commit=True):
 
-class ForumReplyForm(forms.ModelForm):
+class ForumCommentForm(forms.ModelForm):
     class Meta:
-        model = ForumReply
+        model = ForumComment
         fields = ["content"]
+
+class ForumReplyForm(forms.ModelForm):
+    class Meta:
+        model = Reply
+        fields = ["body"]
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)